R/pkg/R/DataFrame.R

```diff
@@ -1125,6 +1125,52 @@ setMethod("summarize",
             agg(x, ...)
           })
 
+#' dapply
+#'
+#' Apply a function to each partition of a DataFrame.
+#'
+#' @param x A DataFrame
+#' @param func A function to be applied to each partition of the DataFrame.
+#' @param schema Optional. The schema of the resulting DataFrame after the function is applied.
+#'               If NULL, the content of the resulting DataFrame is serialized R data.
```

Comment:
Could you say a bit more about what the schema will be, and what operation will be supported?

Since the serialization format is not known to user, how can user use it?